app/assets/stylesheets/home.css.scss

```diff
@@ -10,6 +10,21 @@
   font: 0/0 a; /* Remove the gap between inline(-block) elements */
 }
 
+.new-overlay {
+  /*background: #cc3333;*/
+  opacity: .9;
```

Comment:
`.9` should be written with a leading zero as `0.9`<br>Properties should be ordered color, font-size, opacity, padding, position, top, width